rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -38,6 +39,9 @@
   public static BookKeeper bookKeeper;
   public static BookKeeperService.Processor processor;
 
+  // Registry for gathering & storing necessary metrics
+  private static MetricRegistry metrics = new MetricRegistry();
```

Comment:
Can you make this a member variable that is initialized in the main function or in startServer ?

Reply:
Should we also have a flag to enable metrics ? Just an implementation detail so that we can turn metrics off if anything goes wrong